### PR TITLE
Updated to Angular 13

### LINKED ISSUE #270: Update to Angular 13

### PR DESCRIPTION
**Pull request by MarcelCoding**
**Description**
Based on #291.

**Issue**
Closes #270

**TODO**
- [X] Remove deprications
- [x] Fix Compile Errors 
- [x] Fix tests
- [x] Re-Enable strict null checks
- [x] context menu?? @MarcelCoding Due to 06.03.2022

**SQUASH THIS PR**

### CODE REVIEW

File: src/app/account/account-page-base/account-page-base.component.ts [hunk 1]
```diff
@@ -1,25 +1,63 @@
...
+  particlesLoaded(container: Container): void {
+    console.log(container);
   }
 
-  ngOnDestroy(): void {
-    this.particles.destroy();
+  particlesInit(main: Main): void {
+    console.log(main);
+
+    // Starting from 1.19.0 you can add custom presets or shape here, using the current tsParticles instance (main)
```
Comment (Tristan-H11) [inline src/app/account/account-page-base/account-page-base.component.ts]:
```suggestion
```

File: src/app/account/account-page-base/account-page-base.component.ts [hunk 2]
```diff
@@ -1,25 +1,63 @@
...
 
-  ngOnInit(): void {
-    this.particles = Particles.init({
-      selector: 'canvas',
-      maxParticles: 450,
-      color: '#007f00',
-      speed: 0.5
-    });
+  particlesLoaded(container: Container): void {
+    console.log(container);
```
Comment (Tristan-H11) [inline src/app/account/account-page-base/account-page-base.component.ts]:
```suggestion
```

**Review by Tristan-H11 — CHANGES_REQUESTED** [review]

**Review by Tristan-H11 — DISMISSED** [review]

**Comment by Tristan-H11** [comment]
@MarcelCoding mergeee :D

**Comment by MarcelCoding** [comment]
I don't have the permission to push into master.

**Comment by Tristan-H11** [comment]
@Defelo we want to merge. Can you help us? :)

**Review by Tristan-H11 — DISMISSED** [review]

**Pull request merged**